src/one_dragon/base/controller/pc_button/xbox_button_controller.py

```
@@ -188,18 +220,100 @@ def tap_l_stick_d(self, press: bool = False, press_time: Optional[float] = None)
+        self.pad.update()
+
+    def tap_guide(self, press: bool = False, press_time: float | None = None) -> None:
+        self._press_button(self._btn.XUSB_GAMEPAD_GUIDE, press=press, press_time=press_time)
+
+    def _press_button(self, btn, press: bool = False, press_time: float | None = None):
+        """按下按钮。
+
+        Args:
+            btn: 按键。
+            press: 是否按下。
+            press_time: 按下时间。如果 press=False press_time=None，则使用key_press_time；如果 press=True press_time=None 则不放开。
```

Ruff (0.15.2)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)